scripts/check_compliance.py

```diff
@@ -35,8 +35,10 @@ def error(what):
     '_cwd': os.getcwd()
 }
 
+ZEPHYR_BASE = os.environ.get('ZEPHYR_BASE')
```

Comment:
`os.environ['ZEPHYR_BASE']` would crash if unset instead of returning `None` like `get` does. That's more fail-fast.

Reply:
Almost all the checks work without any ZEPHYR_BASE (the one that doesn't prints "None")

Reply:
That's surprising! Thanks for explaining.